tests/test_smoke.py

```diff
@@ -309,6 +309,22 @@ def test_n_node_job_queue():
     run_one_test(test)
 
 
+def test_large_job_queue():
+    name = _get_cluster_name()
+    test = Test(
+        'job_queue_large',
+        [
+            f'sky launch -y -c {name} --cloud gcp ""',
+            f'for i in `seq 1 75`; do sky exec {name} -d "echo $i; sleep 100000000"; done',
+            f'sky cancel repr 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16',
```

Comment:
repr -> {name}?

Reply:
Oops. Great catch! Thanks!

Reply:
Just tested again with this one and passed.